src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/nomismapping/NomisMappingClient.kt

```diff
@@ -13,6 +13,8 @@ class NomisMappingClient(private val nomisMappingApiWebClient: WebClient) {
     private val log = LoggerFactory.getLogger(this::class.java)
   }
 
+  fun getNomisLocationMappingsBy(dpsLocationIds: Collection<UUID>): Collection<NomisDpsLocationMapping> = dpsLocationIds.mapNotNull { getNomisLocationMappingBy(it) }
```

Comment:
There is no POST endpoint to look up more than one NOMIS mapped location at a time, this is the best we can do for now.